### PR TITLE
Ad 404 page

### PR DESCRIPTION
**Pull request by andrewydai**
2 things to worry about:
- Naming convention (I usede four04 for variables that can't start with number)
- Is using the next library the correct way to reroute to homepage?

### CODE REVIEW

File: utils/courseAPIClient.ts [hunk 1]
```diff
@@ -1,8 +1,161 @@
 import { GraphQLClient } from 'graphql-request';
 import { getSdk } from '../generated/graphql';
 
-const ENDPOINT = 'https://searchneu.com/graphql';
+const ENDPOINT = 'http://localhost:4000';
```
Comment (dajinchu) [inline utils/courseAPIClient.ts]: Just change this back and clean up the big comment block below

**Review by dajinchu — CHANGES_REQUESTED** [review]
looks great. just delete `package-lock.json` and use `yarn` to install dependencies instead of `npm`

File: utils/courseAPIClient.ts (hunk 1 above)
Comment (andrewydai) [reply utils/courseAPIClient.ts]: Thanks! Yeah I think I was just on auto pilot and added some file changes I didn't mean to.

**Review by NEUDitao — APPROVED** [review]

**Comment by NEUDitao** [comment]
can we get this bumped? as much as I love `404: YOU GOT LIGMA!` it'd be cool to see Diana's work in prod

**Review by dajinchu — APPROVED** [review]

**Pull request merged**